ktor-utils/jvm/src/io/ktor/util/Path.kt

```diff
@@ -44,30 +44,49 @@ private fun File.notRooted(): File {
     return File(path.drop(current.name.length).dropWhile { it == '\\' || it == '/' })
 }
 
-private fun File.dropLeadingTopDirs(): File {
+/**
+ * Discards all leading path separators, top dir (..) and current dir (.) references.
+ * The only exception is when the whole path is a current dir reference (.)
+ */
+internal fun dropLeadingTopDirs(path: String): Int {
```

Comment:
Is it possible to make it using `Paths` API? It looks like this code should look much simpler if we use this API.

Reply:
No, we should avoid using Paths API because of Android.